sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -363,6 +364,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled
+Enables or disables web property bag update when DenyAddAndCustomizePages is enabled. When AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled is set to $true, web property bag can be updated even if DenyAddAndCustomizePages is turned on when the user had AddAndCustomizePages (prior to DenyAddAndCustomizePages removing it).
```

Comment:
>DenyAddAndCustomizePages [](http://example.com/codeflow?start=49&length=24)

"Add and Customize pages"

since we're not referring to the param here